Copy edit, first part of README.

Julian-O opened the pull request:
To encourage me to read carefully, I did a copy-edit as I went.

I haven't finished reading yet.

Julian-O committed:
Copy edit, first part of README.

To encourage me to read carefully, I did a copy-edit as I went.

I haven't finished reading yet.

RobertFlatt commented:
Cool, please keep reading.

Julian-O committed:
Copy edit, second part.

Read as far as I am able to, until the tears filled my eyes. (There are *so* many things to go wrong!)

Android-for-Python reviewed (commented):
Looks amazing.

One location where things went astray:
Typos in the first line, and the adb/Buildozer commands seem mixed up.
```
The app is then installed nwith one of these techniqueis:

- If Buildozer was run on a virtual machine such as WSL or Colab copy the `.apk` to a desktop OS, and use adb to install the app (For details on installing and using adb [see Appendix A](#appendix-a--using-adb)):
- Using [Android Debug Bridge](https://developer.android.com/tools/adb) (adb) directly. e.g. 'adb deploy run logcat' will install the app on the device, run it, and display the log.

- Using adb via Buildozer: e.g. `buildozer adb deploy`

- If Buildozer was run on a virtual machine, such as WSL or Colab, copy the `.apk` to a desktop OS, and use adb to install the app (For details on installing and using adb [see Appendix A](#appendix-a--using-adb)):

```

Julian-O committed:
Address review comments

Whoops. I got confused by the various scripts I have been using to automate the installations, so I hadn't actually typed these commands in a long while. Sorry.

Julian-O committed:
More trivial corrections
